### ISSUE TITLE
copy() seems incorrect

### ISSUE THREAD

**Issue by dwieeb** [motemen/gore#37]
Reference: http://play.golang.org/p/_lYNw9SXN5

When trying this example, line by line, gore yields a different result than my test script locally and on the playground.

```
$ gore
gore version 0.2.2  :help for help
gore> a := []string{"hello", "world"}
[]string{"hello", "world"}
gore> b := []string{"goodbye", "world"}
[]string{"goodbye", "world"}
gore> copy(a, b)
2
gore> a
[]string{"hello", "world"}
gore> fmt.Println(a)
[hello world]
14
<nil>
gore>
```

**Comment by motemen** [comment]
Thanks for filing! Fixed in #38.

**Comment by dwieeb** [comment]
Good test! :+1:
Thanks
